pkgs/by-name/re/readest/package.nix

```diff
@@ -19,13 +19,13 @@
 
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "readest";
-  version = "0.9.38";
+  version = "0.9.39";
 
   src = fetchFromGitHub {
     owner = "readest";
     repo = "readest";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-lr1bLNm68jeCjllVWpM07fkRy/PY1C3Ud8XMUQwIef4=";
+    hash = "sha256-1GVWQ5Kckh5od3p1hCfv8vIzCfCY8cRvCBmSZKcF+Qk=";
     fetchSubmodules = true;
   };
 
```

Comment:
This has been bothering me for a while, but I finally figured out how to make TTS work by adding gstreamer and its plugins to the runtime deps:

```diff
   buildInputs = [
     webkitgtk_4_1
     gtk3
     librsvg
     openssl
+    # TTS
+    gst_all_1.gstreamer
+    gst_all_1.gstreamermm
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
   ];
```